applications/salk-portal/src/components/ExplorerSidebar.jsx

```diff
@@ -221,12 +223,16 @@ const Sidebar = (props) => {
           <List component="div" disablePadding>
             <ListItemLink href="#salkteam" onClick={() => props.executeScroll(SALK_TEAM)} selected={hash === `#${SALK_TEAM}`}>
               <ListItemText primary="Salk Institute Team" />
+{/*
```

Comment:
what is the reason to change this? the feature is about fixes this seems no fix to me